Readme.md

```diff
@@ -1,134 +1,25 @@
 <!-- default badges list -->
-![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128577852/21.2.3%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1439)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
-<!-- default file list -->
-*Files to look at*:
+# Pivot Grid for Web Forms - How to Sort a PivotGrid by Clicking on a Field Value
 
-* [Default.aspx](./CS/Default.aspx) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-* [Default.aspx.cs](./CS/Default.aspx.cs) (VB: [Default.aspx.vb](./VB/Default.aspx.vb))
-<!-- default file list end -->
-# How to sort a ASPxPivotGrid by clicking on a field value
-<!-- run online -->
-**[[Run Online]](https://codecentral.devexpress.com/e1439/)**
-<!-- run online end -->
+The PivotGrid allows to sort by any column. To do this, right click any field value and select an appropriate field to sort by. Sometimes it is possible to sort by any column by clicking on it. This example shows how to implement this behavior. 
 
-
-<p>The PivotGrid allows sorting by any column. To do this, right click any field value and select an appropriate field to sort by it. Sometimes it is possible to sort by any column by simply clicking on it. This example shows how to implement this behavior. You should create a template for field values to place a hyperlink (or a button) rather than a plain text.</p>
+Create a template for field values to place a hyperlink (or a button) rather than a plain text. Use the pivot grid's [FieldValueTemplate](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.ASPxPivotGrid.FieldValueTemplate) property.

```

Comment:
`Create a template for field values to place a hyperlink (or a button) rather than a plain text.`

1. What do you mean by "place" in this context? For example, do you mean "display"?

2. Please change "rather than a plain text" to "rather than plain text".

3. `Use the pivot grid's [FieldValueTemplate](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.ASPxPivotGrid.FieldValueTemplate) property.`

Can we update this sentence to the following?

`To create a template, use the pivot grid's [FieldValueTemplate](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.ASPxPivotGrid.FieldValueTemplate) property.`

Reply:
```suggestion
Create a template for field values to display a hyperlink (or a button) instead of plain text. To create a template, use the pivot grid's [FieldValueTemplate](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.ASPxPivotGrid.FieldValueTemplate) property.
```
"Display" seems to be more exact than "place" in this case.